src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/Student.java

```diff
@@ -0,0 +1,39 @@
+package com.github.maleksandrowicz93.cqrsdemo.student;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Entity
+@ToString
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+class Student {
```

Comment:
Jeszcze sprawa warta odnotowania - wszystko robisz na tej klasie, która de facto jest reprezentacją tabeli w DB, a powinno to być mapowane do obiektu domenowego / obiektów domenowych. Tutaj jest to uzasadnione, bo domena jest anemiczna (CRUD), ale generalnie w podejściu "ładnym" (domain driven design) lepiej tego unikać.

Po krótce: https://www.youtube.com/watch?v=CVjdi38gVAo
Przykład z kodem, bo akurat mam pod ręką: https://stackoverflow.com/a/72870633/3305737

Reply:
Jak machnę to, to mogę zrobić ćwiczenie z książkami, żeby faktycznie pokryć problem różnych kontekstów obiektów domenowych.